Index: src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.codecool.dungeoncrawl.logic.actors;\n\nimport com.codecool.dungeoncrawl.logic.Cell;\nimport com.codecool.dungeoncrawl.logic.Drawable;\n\npublic abstract class Actor implements Drawable {\n    private Cell cell;\n    private int health = 10;\n\n    public Actor(Cell cell) {\n        this.cell = cell;\n        this.cell.setActor(this);\n    }\n\n    public void move(int dx, int dy) {\n        Cell nextCell = cell.getNeighbor(dx, dy);\n        cell.setActor(null);\n        nextCell.setActor(this);\n        cell = nextCell;\n    }\n\n    public int getHealth() {\n        return health;\n    }\n\n    public Cell getCell() {\n        return cell;\n    }\n\n    public int getX() {\n        return cell.getX();\n    }\n\n    public int getY() {\n        return cell.getY();\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java b/src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java
--- a/src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java	(revision 0c6a323ccce6843589f3dfce65837289bffdce6e)
+++ b/src/main/java/com/codecool/dungeoncrawl/logic/actors/Actor.java	(date 1673377353405)
@@ -5,7 +5,6 @@
 
 public abstract class Actor implements Drawable {
     private Cell cell;
-    private int health = 10;
 
     public Actor(Cell cell) {
         this.cell = cell;
@@ -20,6 +19,7 @@
     }
 
     public int getHealth() {
+        int health = 10;
         return health;
     }
 
